Week1/homework/style.css

```diff
@@ -1 +1,81 @@
-/* add your styling here */
\ No newline at end of file
+body {
```

Comment:
Nice work with the CSS.  Some really nice extras in here. 👍

Reply:
🙂